packages/experimental/surface/src/plugins/SpacePlugin.tsx

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { useParams } from 'react-router';
+
+import { MulticastObservable } from '@dxos/async';
+import { useSpace } from '@dxos/react-client';
+
+import { Surface, definePlugin, findPlugin } from '../framework';
+import { ClientPluginProvides } from './ClientPlugin';
+import { GraphPluginProvides } from './ListViewPlugin';
+import { RouterPluginProvides } from './RoutesPlugin';
+
+export type SpacePluginProvides = GraphPluginProvides & RouterPluginProvides;
+
+export const SpaceContainer = () => {
+  const { spaceId } = useParams();
+  const space = useSpace(spaceId);
+  return <pre>{JSON.stringify(space?.properties)}</pre>;
+};
+
+export const SpacePlugin = definePlugin<SpacePluginProvides>({
+  meta: {
+    id: 'dxos:SpacePlugin'
+  },
+  provides: {
+    router: {
+      routes: () => [
+        {
+          path: '/space/:spaceId',
+          element: (
+            <Surface
+              component='dxos:SplitViewPlugin/SplitView'
+              surfaces={{
```

Comment:
the nesting seems odd.
shouldn't the embedded component decide about the layout of internal surfaces and rely on plugins to populate that.